### ISSUE TITLE
Experimental Design [Alert: need alt. version of sign-in alerts + lock icon]

### ISSUE THREAD

**Issue by sterkenburgsara** [department-of-veterans-affairs/vets-design-system-documentation#3105]
## What

My HealtheVet has a use case where we need to use a **warning** version of the sign-in alert. While this is possible in Figma, the same flexibility  doesn't exist in the coded instance of the alerts. Developers cannot use the lock icon in the warning alert today, so our team had to create a custom instance of the alert for this purpose. We are hoping that the lock icon can be used in informational and warning alert styles moving forward. 

## Purpose

My HealtheVet needs to have warning alerts with the "lock" icon related to a complicated series of sign-in alerts that help Veterans on the journey to verify their identity and access My HealtheVet. After taking these designs through the Design System Council earlier in the sprint, we were advised to use warning alerts to better indicate that "success" was not imminent. 

## Usage

Same as above

## Behavior

No specific considerations for mobile or other devices

## Examples

[Figma files here](https://www.figma.com/design/m992k2m1DSl9MXV9hDytsQ/MHV-Account-Security-%26-Sign-In?node-id=3-27&t=jyGG6miEI2SHaVS7-0)

<img width="430" alt="Screenshot 2024-07-29 at 4 12 08 PM" src="https://github.com/user-attachments/assets/10525a50-93e2-472a-9ae1-8cc42c391f88">

<img width="671" alt="Screenshot 2024-07-29 at 4 11 12 PM" src="https://github.com/user-attachments/assets/d75c0d94-979f-435c-83b5-2dd36a9583e3">

<img width="147" alt="Screenshot 2024-07-29 at 4 11 28 PM" src="https://github.com/user-attachments/assets/13445b32-d651-4b7d-a03e-cd0583c936c0">

## Accessibility

N/A

## Guidance

For sign-in scenarios where additional steps are necessary in order for a user to unlock or access personal account information. 

## Your team

Cartography Team / My HealtheVet

## Code (optional)

> Include any existing code.

## Next steps

You may present your work to the Design System Council at an upcoming meeting. If you do not or cannot attend the Design Council Meeting, you can opt to get an asynchronous approval.

Submit requests to join an upcoming Design System Council meeting in #platform-design-system.  

During the meeting, the Design System Council Working Group will evaluate the request and make a decision.

If your request is approved, you can [add your component or pattern to the system](https://design.va.gov/about/contributing-to-the-design-system/add-a-component-or-pattern-once-approved). If you have any questions on how to add your component or pattern to the system, please reach out to the Design System Team at #platform-design-system.

**Comment by humancompanion-usds** [comment]
No need to come to DSC. We're already planning on adding additional Sign-in variations as Content Components to the Design System (issues #2556 and #76785) this quarter. We'll add this to the pile of Sign-in alerts.

**Comment by sterkenburgsara** [comment]
Sounds great, thank you so much!

**Comment by humancompanion-usds** [comment]
The new sign-in alerts are available in guidance: https://design.va.gov/components/alert/alert-sign-in/

The new web-component variations of Alert should be in Storybook end of next week and through staging review in the next sprint. We're trying to expedite given a number of teams looking to use these new variations.

When we are done we will close this issue and the 2 predecessors #2284 and #2556

**Comment by humancompanion-usds** [comment]
@sterkenburgsara - Can you use one of the new Alert - Sign-in variations? If not, how about just using the standard Alert - Warning for this? It seems to me like it is going to be time-limited at this point. Thoughts?

**Comment by sterkenburgsara** [comment]
Yes! This is an outdated request - apologies. All of our needs are served by the new sign-in alerts. I'll go ahead and close this ticket. Thank you.
